reference/toaztr.v1.yaml

```diff
@@ -32,126 +32,335 @@ components:
-          tokenUrl: https://auth.toaztr.com/oauth2/token
+          authorizationUrl: 'https://auth.toaztr.com/oauth2/authorize'
+          tokenUrl: 'https://auth.toaztr.com/oauth2/token'
           scopes:
-            simulations/*: Full access to simulations endpoints
-            cases/*: Full access to cases endpoints
+            cases/read: Read-only access to cases
+            cases/write: Write access to cases
+            cases/*: Full access to cases
+            simulations/budget: Access to budget simulations
+            simulations/funding: Access to funding simulations
+            simulations/debt_consolidation: Access to debt consolidation simulations
+            simulations/pinel: Access to pinel simulations
+            simulations/*: Access to all simulations
+            scores/*: Access to all scores
+  responses:
+    '400':
+      description: Bad Request
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    '403':
+      description: Not authorized
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    '404':
+      description: Not Found
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    '409':
+      description: Conflict
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    '500':
+      description: Internal Server Error
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+  parameters:
+    id:
+      name: id
+      in: path
+      required: true
+      schema:
+        type: string
+      description: Unique ID of the resource
+    caseId:
+      name: caseId
+      in: path
+      required: true
+      schema:
+        type: string
+      description: ID of the parent case resource
+    planId:
+      name: planId
+      in: path
+      required: true
+      schema:
+        type: string
+      description: ID of the parent plan-parameters resource
+    IfMatch:
+      name: If-Match
+      in: header
+      required: false
+      schema:
+        type: string
+      description: Entity tag of the resource that is being updated
+    'page[size]':
+      name: 'page[size]'
+      in: query
+      required: false
+      schema:
+        type: number
+        minimum: 0
+        maximum: 20
+        default: 10
+      description: Size of the page to return
+    'page[after]':
+      name: 'page[after]'
+      in: query
+      required: false
+      schema:
+        type: string
+      description: Cursor from which the page starts
+    'page[before]':
+      name: 'page[before]'
+      in: query
+      required: false
+      schema:
+        type: string
+      description: Cursor from which the page starts
```

Comment:
We have a mix of comments in English and in French in this file. Don't you think we need to pass them in French since the descriptions of the objects are in French ?

Reply:
Will fix that

Reply:
done